boards/arm/npcx4m8f_evb/npcx4m8f_evb.dts

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2023 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nuvoton/npcx4m8f.dtsi>
+#include "npcx4m8f_evb-pinctrl.dtsi"
+
+/ {
+	model = "Nuvoton NPCX4M8F evaluation board";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,console = &uart1;
+		zephyr,flash = &flash0;
+		zephyr,keyboard-scan = &kscan_input;
+	};
+
+	aliases {
+		/* For samples/basic/blinky_pwm */
```

Comment:
How attached you are to these comments? :-) I would drop them, it's fairly intuitive from the name what they are for.

Reply:
:-) Let's drop redundant comments.

Reply:
Done.